source/iOS/ObjC-Basic/MM.md

```diff
@@ -400,4 +472,6 @@ myController.completionHandler =  ^(NSInteger result) {
 * https://stackoverflow.com/questions/6055274/use-autorelease-when-setting-a-retain-property-using-dot-syntax
 * https://stackoverflow.com/questions/17601274/arc-and-autorelease
 * https://stackoverflow.com/questions/8292060/arc-equivalent-of-autorelease
-* https://stackoverflow.com/questions/7906804/do-i-set-properties-to-nil-in-dealloc-when-using-arc
\ No newline at end of file
+* https://stackoverflow.com/questions/7906804/do-i-set-properties-to-nil-in-dealloc-when-using-arc
+* [ARC中的Trick](http://ijack.pw/2016/03/17/ARC-naive/)
```

Comment:
默默看亮叔吊打学弟……

> 在 2016年3月22日，上午9:26，辣椒面 <notifications@github.com> 写道：
> 
> In source/iOS/ObjC-Basic/MM.md:
> 
> > \ No newline at end of file
> > +* https://stackoverflow.com/questions/7906804/do-i-set-properties-to-nil-in-dealloc-when-using-arc
> > +* [ARC中的Trick](http://ijack.pw/2016/03/17/ARC-naive/)
> 我靠 亮叔还是起这么早！
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
>